Formula/graphviz.rb

```diff
@@ -58,6 +58,10 @@ def install
     inreplace "lib/sfio/features/sfio", "lib qfrexp\nlib qldexp\n", ""
 
     if build.with? "bindings"
+      # The following setting makes to find "python.h".
+      # https://github.com/Homebrew/homebrew-core/issues/19110
+      ENV["PYTHON_INCLUDES"] = "-I" << `python -c "from __future__ import print_function;import sysconfig; print (sysconfig.get_path('include'))"`.chomp
```

Comment:
Something is wrong here if it cannot find it automatically. This shouldn't be necessary.

Reply:
Without Homebrew's `python`:
```
$ python -c "from __future__ import print_function;import sysconfig; print (sysconfig.get_path('include'))"
/System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7
```
I understand this is too bad.